cmd/soroban-cli/src/print.rs

```diff
@@ -18,21 +18,21 @@ impl Print {
 
     pub fn print<T: Display + Sized>(&self, message: T) {
         if !self.quiet {
-            print!("{message}");
+            eprint!("{message}");
```

Comment:
@fnando I slipped this fix into this PR as well, because it was causing a test to fail where I was adding new usage of the print and that print value was showing up in stdout instead of stderr. I think all this human readable text should be stderr.

Reply:
nice catch!